client/schemes/jellyfinScheme.ts

```diff
@@ -153,16 +165,14 @@ export default class JellyfinScheme {
     }
 
     // Reset everything
-    return this.$auth.reset();
+    this.$auth.reset();
   }
 
-  reset(): Promise<void> {
```

Comment:
Don't change the standard stuff used by Nuxt Auth, please.

If it returns a Promise in the local scheme ([which it does](https://github.com/nuxt-community/auth-module/blob/master/lib/schemes/local.js#L114) it should return one here.

That makes it easier to reference back to the original and update it when/if necessary.